pkg/cmd/roachtest/github.go

```diff
@@ -80,7 +102,7 @@ func (g *githubIssues) createPostRequest(t test.Test, message string) issues.Pos
 		branch = "<unknown branch>"
 	}
 
-	artifacts := fmt.Sprintf("/%s", t.Name())
+	artifacts := fmt.Sprintf("/%s", issueName)
```

Comment:
I think this is going to break the artifacts link on the GitHub issues. The artifacts path will still be under the original test's name.

Reply:
that's a good catch - it would have been breaking on master too but we only override for cluster creation there